components/Order/index.rst

```diff
@@ -4,10 +4,12 @@ Order
 E-Commerce PHP library for creating and managing sales orders.
 
 .. toctree::
-   :numbered:
+   :maxdepth: 2
 
    installation
-   order_and_items
-   adjustments
+   models
+   interfaces
    order_states
-   summary
+   order_events
+   order_transitions
+   basic_usage
```

Comment:
The thing is, that I'd need to go through all these state_machine, models, interfaces etc., which give insights about how small parts works, but I am convinced that when you read about component, you want to know what it can do as soon as possible.